opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -391,18 +401,24 @@ message HistogramDataPoint {
   // The set of labels that uniquely identify this timeseries.
   repeated opentelemetry.proto.common.v1.StringKeyValue labels = 1;
 
-  // start_time_unix_nano is the time when the cumulative value was reset to zero.
+  // start_time_unix_nano is the last time when the value was reset to "zero".
```

Comment:
```suggestion
  // start_time_unix_nano is the last time when the aggregation value was reset to "zero".
```

Reply:
I think it's a little weird to say `value was reset to "zero"` for a Histogram/Summary but I know that's why you quoted it.

I would also be in favor of changing all of the comments to `start_time_unix_nano is the last time when the aggregation was reset`.

Reply:
done.